application/gqlgen/schema.graphql

```diff
@@ -140,6 +142,20 @@ type Post {
     isEditable: Boolean!
 }
 
+type Meeting {
+    id: ID!
+    name: String!
+    description: String
+    moreInfoURL: String
+    startDate: Time!
```

Comment:
When post had dates, we used `String` because `Time` returns a full ISO date-time string. Maybe this won't be a problem since this is a new field that the UI hasn't implemented yet.  On the other hand, there may be a risk of time zone conversion problems if time is included. @wcjr do you have any input here?

Reply:
afaik we don't need time at all, I'd be inclined to keep it as a String with a date.

Reply:
@wcjr , sorry, I should have given you more context. The internal implementation is date-only, so if the gql type is `Time` you'll get something like "2019-12-17T00:00:00Z" as opposed to "2019-12-17" if the type is `String`.

Reply:
I'd prefer the `String` so we just get the date.

Reply:
Done